component-maps.yml

```diff
@@ -4,6 +4,13 @@
 # and "docker_container". Binary tools that don't have Docker components, will
 # only have the "git" part.
 git:
+  azure-iot-manager:
+    docker_image:
+    - azure-iot-manager
+    docker_container:
+    - mender-azure-iot-manager
+    release_component: false
```

Comment:
This is missing the corresponding `docker_image` and `docker_container` entries.